server/server.js

```diff
@@ -148,6 +148,24 @@ module.exports = function(port, tweetSearcher, googleAuthoriser) {
         }
     });
 
+    app.put("/admin/administrators", function(req, res) {
+        try {
+            googleAuthoriser.addAdmin(req.body.email);
+            res.sendStatus(200);
+        } catch (err) {
+            res.sendStatus(404);
```

Comment:
and also if you throw an error inside a promise block, that promise will reject with the argument given to throw, i.e. throw 500; will pass 500 to the catch